docs/main_text.md

```diff
@@ -76,11 +81,15 @@ If you would prefer to report issues via e-mail, you can also email [ccdl@alexsl
 
 ## refine.bio processed
 
```

Comment:
What do you think of placing the badge next to the title for refine.bio processed and submitter-processed?

![title-badge](https://user-images.githubusercontent.com/15315514/44591179-0faba500-a78b-11e8-9b86-27f6d5634d90.png)